syft/workers/utils.py

```diff
@@ -0,0 +1,41 @@
+import syft as sy
+import random
+from .plan import Plan
+
+
+def make_plan(plan_blueprint):
+    """For folks who would prefer to not use a decorator, they can use this function"""
+    return func2plan()(plan_blueprint)
+
+
+def func2plan():
```

Comment:
You can write code that functions as either a decorator or just a normal function. We should do that here rather than having two functions